workers/generic-worker/usage.go

```diff
@@ -32,15 +32,19 @@ and reports back results to the queue.
 
   Usage:
     generic-worker run                      [--config         CONFIG-FILE]
+                                            [--with-worker-runner]
                                             [--worker-runner-protocol-pipe PIPE]
                                             [--configure-for-aws | --configure-for-gcp | --configure-for-azure]` + installServiceSummary() + `
+
```

Comment:
I think this blank line broke docopt which indirectly caused the Windows failures. The broken tests on windows are calling a custom windows target which is defined in `customTargetsSummary()` on line 38. docopt doesn't see those options due to the blank line here, so returns the usage text.